docs/tutorials/working-with-llms.md

```diff
@@ -0,0 +1,227 @@
+
+#### Knowledge Collections
+You can create knowledge collections by simply uploading documents containing your custom knowledge. You can access the Knowledge Collections management page via `Workspace > Knowledge`.
+
+![](../img/working-with-open-webui/knowedge-collections.png)
+
+#### Web Search
+
+In addition to using local knowledge collections, Open WebUI allows you to perform LIVE web searches to retrieve relevant information to generate better responses. You can use a variety of web search providers as listed in the [Open WebUI's documentation](https://docs.openwebui.com/tutorials/features/web_search).
+
+Rancher Desktop's Open WebUI extension installs and configures [SearXNG](https://docs.searxng.org/), a free internet metasearch engine, to enable you use web search out of the box. 
+
+#### Using Knowledge Collections and/or Web Search in chats
+
+To use a Knowledge Collection in the chat, simply type `#` in the chat input field which shows the available knowledge collections on your machine. You can pick one or more knowledge collections as knowledge source(s) to retrieve relevant context.
+
+:::tip
+You can also include a web page's content in your conversations using the `#` command followed by the web page's URL.
+:::
+
+![](../img/working-with-open-webui/selecting-knowedge-collections.png)
+
+To use the web search in the chat, click the `+` button to the left of the chat input field and enable the web search feature. This allows the chat to perform a web search to get relevant information and generate responses.
+
+![](../img/working-with-open-webui/enabling-web-search.png)
+
+:::tip
+You can use both Knowledge Collections and Web Search to augment the LLM's response to a single question.
+:::
+
+:::info
+Open WebUI provides advanced configuration to fine-tune the Retrieval Augmented Generation process, which allows you to use alternate embedding models, provide a more suitable prompt template, configure the chunk parameters, and much more. You can perform this advanced configuration on the `Admin Panel > Settings > Documents` page. To learn more, refer to the [Open WebUI's Retrieval Augmented Generation documentation](https://docs.openwebui.com/tutorials/features/rag).
+:::
+
+### Customizing Models
+
+Open WebUI provides features to let you customize the downloaded models for your specific domain, use cases, etc. You can customize models on the `Workspace > Models` page. You can create a model by extending one of the existing models. You can customize the models by tweaking the chat parameters, including knowledge collections, etc. The custom models created here can then be used with the Chat and Playground features.
+
+![](../img/working-with-open-webui/customize-models.png)
+
+## Uninstalling the extension
+
+You can uninstall the Open WebUI extension via the [Extensions Catalog](../ui/extensions.md#catalog) GUI or using the [rdctl CLI](../references/rdctl-command-reference.md#rdctl-extension-install).
```

Comment:
I thought if someone has a scripted setup to install, launch, uninstall RD and if they want to install/uninstall extensions then the `rdctl` way is still useful. I agree that not many would use scripts to install/uninstall RD. 

Since the `rdctl` way is in a separate tab that is not default, I will leave it there for now as it doesn't harm to have that additional piece of info there.